src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
```

Comment:
If you create a formatter, make it a constant field. This DateTimeFormatter is used multiple times in your code, so it should be extracted to a constant field.